Documentation/BasicUsage.md

```diff
@@ -0,0 +1,267 @@
+# Basic Usage
+
+For starters we're going to import everything ready to use.  
+Most of the included functionality is similar to how you would deal with WebRTC in your browser.  
+We support most of the official WebRTC APIs, see this [document](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) for more details.
+
+```javascript
+import {
+	ScreenCapturePickerView,
+	RTCPeerConnection,
+	RTCIceCandidate,
+	RTCSessionDescription,
+	RTCView,
+	MediaStream,
+	MediaStreamTrack,
+	mediaDevices,
+	permissions,
+	registerGlobals
+} from 'react-native-webrtc';
+```
+
+## Defining Media Constraints
+
+```javascript
+var mediaConstraints = {
+	audio: {
+		googEchoCancellation: true,
+		googAutoGainControl: true,
+		googNoiseSuppression: true,
+		googHighpassFilter: true,
+		googEchoCancellation2: true,
+		googAutoGainControl2: true,
+		googNoiseSuppression2: true
+	},
+	video: {
+		mandatory: {
+			minFrameRate: '30'
+		},
+		facingMode: 'user',
+		optional: []
+	}
+};
+```
+
+## Getting a Media Stream
+
+```javascript
+var localMediaStream;
+var isVoiceOnly: boolean = false;
```

Comment:
why the type? we don't declare any types in the examples.

Reply:
I was going to add types to everything based on Flow but not quite finished going over everything.
Would you prefer with or without types?